js/src/cleantalk-comments-checkspam.js

```diff
@@ -173,9 +173,9 @@ function ctSendComments() {
                     let statusString = String(ctCommentsCheck.ct_status_string);
                     stastatusStringtusString = statusString.printf(ctCommentsChecked, ctCommentsSpam, ctCommentsBad);
```

Comment:
Here is the mistypo. Just rename `stastatusStringtusString` to `statusString` here and you can use this variable name in the 176 and 178 lines.